aliases/install.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
its used by the update command, but its also the file that is referenced by the first installation command in the readme